sandbox.go

```diff
@@ -767,10 +774,6 @@ func (sb *sandbox) restoreOslSandbox() error {
 		if len(i.llAddrs) != 0 {
 			ifaceOptions = append(ifaceOptions, sb.osSbox.InterfaceOptions().LinkLocalAddresses(i.llAddrs))
 		}
-		if len(ep.virtualIP) != 0 {
-			vipAlias := &net.IPNet{IP: ep.virtualIP, Mask: net.CIDRMask(32, 32)}
-			ifaceOptions = append(ifaceOptions, sb.osSbox.InterfaceOptions().IPAliases([]*net.IPNet{vipAlias}))
-		}
```

Comment:
With the loopback interface replacing the endpoint  interface to host the virtual-ip, there is no need for `restoreOslSandbox` to restore the states, since there is no state to be restored in OSL Sandbox :-)
The change is good to go as is.